### PR TITLE
Improve informations on community page

### PR DESCRIPTION
**Pull request by Czaki**
# References and relevant issues

# Description

When with @melonora, we have preparing a presentation on EuroScipy we found that it will be nice to have single code that could be provided to listeners with links to all our places. 

After checking our page, it looks like community page is best for this. 

This PR is improving the content of this page by adding links to missing places and fixing contributing links. 
I have also added headers to improve readability.

### CODE REVIEW

**Comment by Czaki** [comment]
The inspiration for this PR was this slide from @jni presentation:

Screenshot 2024-08-21 at 13 58 52

I may work on improving how the page looks, but it may require to embed some HTML code.

**Comment by Czaki** [comment]
I do not understand why docs build failed as preview link works

File: docs/community/index.md [hunk 1]
```diff
@@ -5,13 +5,32 @@ There are several different ways to be a part of the napari community. From
...
+
+## Contributing and Bug reporting
+
+The napari development happens in napari repository [github.com/napari/napari](https://github.com/napari/napari). 
+If you spot a bug, please check our [issues list](https://github.com/napari/napari/issues) and if you 
+do not found matching report pleas open new issue. 
+
 If you are interested in contributing, check out our
-[contributing guide](contributing).
+[contributing guide](napari-contributin).
```
Comment (melissawm) [inline docs/community/index.md]:
```suggestion
[contributing guide](napari-contributing).
```

This is why the docs build is failing, see https://app.circleci.com/pipelines/github/Czaki/napari-docs/110/workflows/1a59d4a2-ea8b-46ea-8d92-b8911748d94e/jobs/110?invite=true#step-106-128362_126

File: docs/community/index.md [hunk 2]
```diff
@@ -5,13 +5,32 @@ There are several different ways to be a part of the napari community. From
...
 examples, you are welcome to help develop and improve napari.
 
+## Questions
+
 To ask questions and interact with the team, you can join our
 [zulip chat](https://napari.zulipchat.com/login/) or participate in our
 [meetings](meeting-schedule).
 
+To ask general question about napari usage to whole community of napari user, also for plugins 
+You may visit ![forum image.sc logo](../images/image_sc_logo.png) [forum.image.sc/tag/napari](https://forum.image.sc/tag/napari)
```
Comment (melissawm) [inline docs/community/index.md]:
```suggestion
To ask general questions about napari usage to the community, including plugins,
you can visit ![forum image.sc logo](../images/image_sc_logo.png) [forum.image.sc/tag/napari](https://forum.image.sc/tag/napari)
```

File: docs/community/index.md [hunk 3]
```diff
@@ -5,13 +5,32 @@ There are several different ways to be a part of the napari community. From
...
+To trace latest news from napari you may follow us on:
+ * ![twitter logo](../images/twitter_logo.svg) twitter/X [napari_imaging](https://x.com/napari_imaging/) 
+ * ![mastodon logo](../images/mastodon_logo.svg) mastodon [fosstodon.org/@napari](https://fosstodon.org/@napari)    
+
+
+## Contributing and Bug reporting
+
+The napari development happens in napari repository [github.com/napari/napari](https://github.com/napari/napari). 
+If you spot a bug, please check our [issues list](https://github.com/napari/napari/issues) and if you 
+do not found matching report pleas open new issue.
```
Comment (melissawm) [inline docs/community/index.md]:
```suggestion
The napari development happens in the napari GitHub repository [github.com/napari/napari](https://github.com/napari/napari). 
If you spot a bug, please check our [issues list](https://github.com/napari/napari/issues) . If you 
do not find a matching report, please open a new issue. 
```

**Review by melissawm — COMMENTED** [review]
Left a couple suggestions for the wording, otherwise I'm +1 on the idea! Thanks @Czaki

**Comment by melissawm** [comment]
Tiny nitpick, but we may want to make the icons a bit smaller? At this size, the spacing and surrounding text are a bit weird to me. Not blocking, though!

Captura de imagem_20240821_094234

**Comment by Czaki** [comment]
They may be smaller. For SVG it is simple. The main question is "how smaller"

**Comment by melissawm** [comment]
How about something like this? I used 25px as the image height:

Captura de imagem_20240821_102215

On mobile:

Captura de imagem_20240821_102350

**Review by melissawm — APPROVED** [review]
Looks good to me. Unfortunately the imagesc logo got broken from the link which is in the subsequent line - maybe we can add a couple of words to that sentence to avoid it? I'm approving either way as I don't think that's a real problem just a tiny detail 😄

**Comment by psobolewskiPhD** [comment]
Do y'all think non-breaking spaces between the logos and links would work?
`&nbsp;`?
(Poor zulip, no logo 😢)

**Review by psobolewskiPhD — APPROVED** [review]

**Pull request merged**